<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgSettingsAudio</class>
 <widget class="QWidget" name="wdgSettingsAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>580</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">wdgSettingsAudio</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_wdgSettingsAudio">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_Audio">
     <item row="10" column="0" colspan="4">
      <widget class="QCheckBox" name="checkBox_Swap_Duty_Cycles">
       <property name="text">
        <string>Swap Duty Cycles</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSlider" name="horizontalSlider_Buffer_Size_factor">
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_Sample_Rate">
       <property name="text">
        <string>Sample Rate</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4">
      <widget class="QCheckBox" name="checkBox_Reverse_bits_DPCM">
       <property name="text">
        <string>Reverse bits of DPCM</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="icon_Channels">
       <property name="text">
        <string notr="true">Channels icon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QComboBox" name="comboBox_Output_Devices">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="wdgAPUChannels" name="widget_APU_Channels" native="true">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_Output_Devices">
       <property name="text">
        <string>Output Devices</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_Channels_Delay">
       <property name="text">
        <string>Channels Delay</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="icon_Output_Devices">
       <property name="text">
        <string notr="true">Output Devices icon</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="4">
      <spacer name="verticalSpacer_Misc">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>526</width>
         <height>174</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QWidget" name="widget_Samplarate" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_Samplarate">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Samplarate_192000">
          <property name="text">
           <string>192000</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Samplarate_96000">
          <property name="text">
           <string>96000</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Samplarate_48000">
          <property name="text">
           <string>48000</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Samplarate_44100">
          <property name="text">
           <string>44100</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Samplarate_22050">
          <property name="text">
           <string>22050</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Samplarate_11025">
          <property name="text">
           <string>11025</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QWidget" name="widget_Channels" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_Channels">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Channels_Mono">
          <property name="text">
           <string>Mono</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Channels_Stereo_Delay">
          <property name="text">
           <string>Stereo Delay</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Channels_Stereo_Panning">
          <property name="text">
           <string>Stereo Panning</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_Buffer_Size_factor_value">
       <property name="text">
        <string notr="true">10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_APU_Channels">
       <item>
        <widget class="QLabel" name="icon_APU_Channels">
         <property name="text">
          <string notr="true">APU Channels icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_APU_Channels">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>APU Channels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_APU_Channels">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_Channels_Delay_value">
       <property name="text">
        <string notr="true">100%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="QCheckBox" name="checkBox_Enable_Audio">
       <property name="text">
        <string>Enable Audio</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_Audio_misc">
       <item>
        <widget class="QLabel" name="icon_Audio_misc">
         <property name="text">
          <string notr="true">Misc icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Audio_misc">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Misc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_Audio_misc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_Audio_settings">
       <item>
        <widget class="QLabel" name="icon_Audio_settings">
         <property name="text">
          <string notr="true">Settings icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Audio_settings">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_Audio_settings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="icon_Sample_Rate">
       <property name="text">
        <string notr="true">Sample Rate icon</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_Buffer_Size_factor">
       <property name="text">
        <string>Buffer Size factor</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="icon_Channels_Delay">
       <property name="text">
        <string notr="true">Channels Delay icon</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_Channels">
       <property name="text">
        <string>Channels</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="icon_Buffer_Size_factor">
       <property name="text">
        <string notr="true">Buffer Size factor icon</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSlider" name="horizontalSlider_Channels_Delay">
       <property name="maximum">
        <number>19</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgAPUChannels</class>
   <extends>QWidget</extends>
   <header>wdgAPUChannels.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox_Output_Devices</tabstop>
  <tabstop>pushButton_Samplarate_48000</tabstop>
  <tabstop>pushButton_Samplarate_44100</tabstop>
  <tabstop>pushButton_Samplarate_22050</tabstop>
  <tabstop>pushButton_Samplarate_11025</tabstop>
  <tabstop>horizontalSlider_Buffer_Size_factor</tabstop>
  <tabstop>pushButton_Channels_Mono</tabstop>
  <tabstop>pushButton_Channels_Stereo_Delay</tabstop>
  <tabstop>pushButton_Channels_Stereo_Panning</tabstop>
  <tabstop>horizontalSlider_Channels_Delay</tabstop>
  <tabstop>widget_APU_Channels</tabstop>
  <tabstop>checkBox_Reverse_bits_DPCM</tabstop>
  <tabstop>checkBox_Swap_Duty_Cycles</tabstop>
  <tabstop>checkBox_Enable_Audio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
